recipes/ddocent/version.patch

```diff
@@ -0,0 +1,17 @@
+# fix samtools version check
+# (as is latest dev verson of ddocent which is unfortunatelly not released yet)
+diff -ruN dDocent-2.7.8/dDocent dDocent-2.7.8/dDocent
+--- dDocent-2.7.8/dDocent	2019-05-02 20:51:03.000000000 +0200
++++ dDocent-2.7.8/dDocent	2020-11-03 11:07:17.153106953 +0100
+@@ -28,8 +28,8 @@
+ 	fi
+ done
+ 
+-SAMV1=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -c1)
+-SAMV2=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -c3)
++SAMV1=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -f1 -d ".")
++SAMV2=$(samtools 2>&1 >/dev/null | grep Ver | sed -e 's/Version://' | cut -f2 -d " " | sed -e 's/-.*//' | cut -f2 -d ".")
++echo "SAMV1 $SAMV1 SAMV2 $SAMV2"
```

Comment:
I assume this is just for debugging purposes in conjunction with the `samtools --version` command in the tests and both will be removed before merging this PR, right?

Reply:
(With "this" I mean the `echo`.)

Reply:
Jep. Thanks.